neuromation/cli/config.py

```diff
@@ -28,6 +29,11 @@ def config() -> None:
     """Client configuration."""
 
 
+@group()
+def quota() -> None:
```

Comment:
Don't use nested group, `neuro config show-quota` is just fine.

Reply:
AFAIK, the team decided to use `neuro config quota show` in order to extend `quota` group with additional funcitonality: `neuro config quota add`, etc. Perhaps, the good solution would be to make the group `quota` non-nested: `neuro quota show`

Reply:
We decided to put quota management into the config group, that's it.
Please don't add redundant nesting.

Reply:
I think the common user cannot change quota.

Reply:
Also please keep in mind: later, after organizations implementing, we may find that quota better fits somewhere in the organization management structures. Now we don't add top-level `neuro quota` but put a simple command into the config group for this reason.

Reply:
OK